models/cuboid/Cargo.toml

```diff
@@ -3,5 +3,11 @@ name = "cuboid"
 version = "0.1.0"
 edition = "2021"
 
-[dependencies.fj]
-path = "../../crates/fj"
+[[bin]]
+name = "cuboid"
+path = "src/main.rs"
```

Comment:
Why add this? This is the default behavior anyway, and I don't see a benefit in having it.